repo/templates/.github/workflows/check-pr.yml

```diff
@@ -0,0 +1,22 @@
+name: Check PR
+
+on: pull_request
+permissions: read-all
+
+concurrency:
+  group: check-pr-$\{{ github.head_ref || github.ref_name }}
```

Comment:
Why do I get 
```
✖  +! Parse error on line 12:
...${{ github.head_ref || github.ref_name }
-----------------------^
Expecting 'CLOSE_RAW_BLOCK', 'CLOSE', 'CLOSE_U```

Reply:
Good spot! It was a typo - the curly braces were not correctly escaped. I've fixed that now